logback/build.gradle

```diff
@@ -5,5 +5,7 @@ dependencies {
         testImplementation project(':thrift0.17')
     }
     // Logback
+    compileOnly libs.logback14
+    testCompileOnly libs.logback14
```

Comment:
I'm unsure if `logback14` dependency in `logback` module is maintainable.
It might be difficult to know which method belongs to which version statically.

What do you think of creating `logback13` module instead? We may copy most sources using Gradle scripts and selectively override some files for Logback 1.3 like Spring Boot or Thrift modules.

Reply:
@ikhoon 
Thanks for your suggestions.
I added the `logback13` modules. (I saw `junit4` modules as a reference) 
Please tell me if my changes are not good.